docs/elasticsearch/cluster.md

```diff
@@ -198,6 +189,135 @@ x.x.x.x           23          38   0    0.12    0.07     0.06 md        -      o
 
 To better understand and monitor your cluster, use the [cat API](../catapis/).
 
+## Step 6: Configure shard allocation awareness or forced awareness
+
+If your nodes are spread across many geographical zones, you can configure shard allocation awareness to allocate all replica shards to a zone that’s different from their primary shard.
+
+With shard allocation awareness, if the nodes in one of your zones fail, you can be assured that your replica shards are spread across your other zones. It adds a layer of fault tolerance than just individual node failure.
+
+To configure shard allocation awareness, add zone attributes to `odfe-d1` and `odfe-d2`, respectively:
+
+```yml
+node.attr.zone: 1
+```
+```yml
+node.attr.zone: 2
+```
+
+Log in to your master node and update the cluster settings:
+
+```json
+PUT _cluster/settings
+{
+  "persistent": {
+    "cluster.routing.allocation.awareness.attributes": "zone"
+  }
+}
+```
+
+You can either use `persistent` or `transient` settings. Persistent settings persist through a cluster reboot, transient settings do not.
+
+Shard allocation awareness prefers to separate primary and replica shards across multiple zones but if there is only one zone available like in the event of a failure, then replica shards are allocated to the only remaining zone.
+
+You also have the option to require that primary and replica shards are never allocated to the same zone. This is called forced awareness.
+
+To configure forced awareness, specify all the possible values for your zone attributes:
+
+```json
+PUT _cluster/settings
+{
+  "persistent": {
+    "cluster.routing.allocation.awareness.attributes": "zone",
+    "cluster.routing.allocation.awareness.force.zone.values":["1", "2"]
+  }
+}
+```
+Now, if one of the data nodes fail, forced awareness will not allocate the replicas to a node in the same zone. Instead, it will keep the cluster in a yellow state and only allocate replicas when nodes in another zone come back online.
+
+You generally want to use allocation awareness when the hardware in each of your zones is less than 50% utilized so that it has the storage capacity to allocate replicas in the same zone.
+
+If you’re in a situation where you do not have the capacity in any one zone to contain the entire cluster with all its primary and replica shards, then you should use forced awareness. So in the event of a failure, you don't over-allocate your last remaining zone and essentially lock up your cluster because it's storage is maxed out.
+
+## Step 7: Set up a hot/warm architecture
+
+You can design a hot/warm architecture where you first index your data to hot nodes---fast and expensive---and after a certain period of time move them to warm nodes---slow and cheap.
```

Comment:
Definitely need more of an explanation for the use case here. General stuff:

- Log analytics: if you have a bunch of data you're streaming in, rarely update, and want the older stuff to go onto cheaper storage, this can be a good fit.
- Search: not a great fit, because data never really "ages out" and the indices are frequently updated.

Reply:
How about "If you analyze time-series data that you rarely update and want the older data to go onto a cheaper storage, this architecture can be a good fit"?

Reply:
I don't think "time series" needs the hyphen, but otherwise, sounds great.